src/Proto.Cluster/Identity/IdentityStoragePlacementActor.cs

```diff
@@ -57,8 +71,13 @@ private Task Tick(IContext context)
             return Task.CompletedTask;
         }
 
-        private async Task Terminated(Terminated msg)
+        private async Task Terminated(IContext context, Terminated msg)
         {
+            if (context.System.Token.IsCancellationRequested)
+            {
+                return;
```

Comment:
silent exit on shutdown.

This one was also tricky.
When the placement actor is shutting down, e.g. from IdentityLookup.ShutdownAsync.
it also kills all of its children. which is what parent actors do.

BUT, each child was sending terminated events which we acted on and removed activation entries on